internal/reconcilers/anchor.go

```diff
@@ -369,18 +364,33 @@ func (r *AnchorReconciler) getNamespace(ctx context.Context, nm string) (*corev1
 	return ns, nil
 }
 
-func (r *AnchorReconciler) writeNamespace(ctx context.Context, log logr.Logger, nm, pnm string) error {
-	inst := &corev1.Namespace{}
+func (r *AnchorReconciler) updateNamespace(ctx context.Context, log logr.Logger, nm, pnm string) error {
+	pnmInst := &corev1.Namespace{}
+	pnnm := types.NamespacedName{Name: pnm}
+	if err := r.Get(ctx, pnnm, pnmInst); err != nil {
+		return err
+	}
+
+	inst := &corev1.Namespace{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Namespace",
+			APIVersion: corev1.SchemeGroupVersion.String(),
+		},
+	}
 	inst.ObjectMeta.Name = nm
 	metadata.SetAnnotation(inst, api.SubnamespaceOf, pnm)
+	for k, v := range pnmInst.Labels {
+		if strings.Contains(k, api.MetaGroup) {
+			continue
+		}
+		metadata.SetLabel(inst, k, v)
+	}
 
-	// It's safe to use create here since if the namespace is created by someone
-	// else while this reconciler is running, returning an error will trigger a
-	// retry. The reconciler will set the 'Conflict' state instead of recreating
-	// this namespace. All other transient problems should trigger a retry too.
-	log.Info("Creating subnamespace")
-	if err := r.Create(ctx, inst); err != nil {
-		log.Error(err, "While creating subnamespace")
+	log.Info("Updating subnamespace")
+	if err := r.Patch(ctx, inst, client.Apply, &client.PatchOptions{
+		FieldManager: api.MetaGroup,
+	}); err != nil {
```

Comment:
A couple of things here:
* This was a very important comment, are you sure it's safe to delete?
* Where does the subnamespace get created if you've taken away the "Create" call?
* We use "r.Update" everywhere in HNC, why are you using "r.Patch" here? If this is important, there should be a very clear comment explaining why, otherwise someone will "fix" it in the future.